.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: deno upgrade --canary
 
       - name: Format
-        run: deno fmt --check
+        run: deno fmt --check --ignore=fs/testdata/json_invalid.json
```

Comment:
Again, this should be removed and inline ignore should be added to the file

Reply:
How does that work?

Reply:
Add `// deno-fmt-ignore-file` in the appropriate file

Reply:
but it's json..

Reply:
Right... This is unlucky, we could fix that by changing it to JSONC